.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Build and upload to PyPI
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.ref }}
+  cancel-in-progress: true
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+
+  build_sdist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    outputs:
+      SDIST_NAME: ${{ steps.sdist.outputs.SDIST_NAME }}
```

Comment:
```suggestion
```
I don't think we use that here. It is only used in those other examples when combining sdists and wheels, but it looks like you are just doing sdist and avoiding needing this.